static/app/views/starfish/components/tableCells/transactionIdCell.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'sentry/components/links/link';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+
+interface Props {
+  orgSlug: string;
+  projectSlug: string;
+  transactionId: string;
+  spanId?: string;
+}
+
+export function TransactionIdCell({projectSlug, orgSlug, transactionId, spanId}: Props) {
+  let url = normalizeUrl(
+    `/organizations/${orgSlug}/performance/${projectSlug}:${transactionId}`
+  );
+
+  if (spanId) {
+    url += `#span-${spanId}`;
+  }
+
+  return <Link to={url}>{transactionId.slice(0, 8)}</Link>;
```

Comment:
l: I know we don't do this for performance pages or for any of the modules (and that interactions are coming in future PR), but what do you think about using https://github.com/getsentry/sentry/blob/master/static/app/views/discover/table/quickContext/quickContextHovercard.tsx here?

<img width="350" alt="image" src="https://github.com/getsentry/sentry/assets/18689448/c939f1c8-7967-4335-89a4-11b0bebcacc2">

If it works well, we should also add to queries, resources, web vitals, etc.

Reply:
I like this! Added to my backlog